README.md

```diff
@@ -194,6 +194,16 @@ of more than one.
 
 `bundle exec rake`
 
+### Testing in a browser
+
+If you want to test this behaviour in a browser without Fastly set up (in dev
```

Comment:
I swear it didn't work for me the other day, although that was probably because I'd made a mistake somewhere else.  It works now!  I'll change the wording anyway.  Thanks both